src/platform/Tizen/WiFiManager.cpp

```diff
@@ -694,6 +924,33 @@ CHIP_ERROR WiFiManager::Disconnect(const char * ssid)
     return err;
 }
 
+CHIP_ERROR WiFiManager::StartWiFiScan(ByteSpan ssid, DeviceLayer::NetworkCommissioning::WiFiDriver::ScanCallback * callback)
+{
+    // If there is another ongoing scan request, reject the new one.
+    VerifyOrReturnError(sInstance.mpScanCallback == nullptr, CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrReturnError(ssid.size() <= sizeof(sInstance.mInterestedSSID) - 1, CHIP_ERROR_INVALID_ARGUMENT);
+
+    CHIP_ERROR err       = CHIP_NO_ERROR;
+    int wifiErr          = WIFI_MANAGER_ERROR_NONE;
+    bool isWiFiActivated = false;
+
+    memcpy(sInstance.mInterestedSSID, ssid.data(), sizeof(sInstance.mInterestedSSID));
```

Comment:
This is a buffer overrun.

Reply:
I have changed copy size from sizeof(sInstance.mInterestedSSID)  to ssid.size(). Due to check at line 931, it will not overflow the destination buffer. 
Please check.

Reply:
Yes, that makes a lot more sense.